catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -833,6 +836,24 @@ private void setProxyForTerrain(Map terrainProvider) {
     return config;
   }
 
+  public void setIconConfig(List<String> icons) {
+    Map<String, Map<String, String>> iconMap = new HashMap<>();
+
+    icons.forEach(
+        icon -> {
+          String key = icon.split("=")[0];
+          String[] config = icon.split("=")[1].split(",");
+          Map<String, String> tempMap = new HashMap<>();
+          tempMap.put("className", config[0]);
```

Comment:
should we be checking the size of `config` after the split?

Reply:
I can add that

Reply:
https://github.com/codice/ddf/pull/5715/commits/700e08948477f64fb6261225cea6847283d2ceaa Good enough?